locations/templates/quarkus-application/template.yaml

```diff
@@ -175,17 +175,32 @@ spec:
             host:
               title: Repo Host
               type: string
-              default: github.com
+              default: gitea.cnoe.localtest.me:8443
+              enum:
+                - gitea.cnoe.localtest.me:8443
```

Comment:
Same as above.

Reply:
When the job will work as github workflow, I will clean the template of this PR. Please wait

Reply:
First enum value is now
```
              enum:
                - github.com
                ...
```